src/grpc/packet_router/hpr_packet_down.erl

```diff
@@ -91,7 +91,7 @@ window(WindowMap) ->
 window(TS, FrequencyHz, DataRate) ->
     WindowMap = #{
         timestamp => TS,
-        frequency => FrequencyHz,
+        frequency => round(FrequencyHz),
```

Comment:
Since `round()` drops precision, please add a comment as to *why* that loss in precision is necessary-- here and the other location (below) where it gets called

Reply:
I will add a comment.